src/store/styleguide/property/object-property.ts

```diff
@@ -11,7 +13,9 @@ export class ObjectProperty extends Property {
 	/**
 	 * The nested properties this property supports in its object values.
 	 */
-	private properties: Map<string, Property> = new Map();
+	private _properties?: Map<string, Property>;
```

Comment:
We never used underscore prefixes before and I'd like to vote against them.
Properties should always be private, and their access by getters/setters, so we don't need to distinguish any privacy here.